src/main/java/seedu/address/logic/parser/SearchParser.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.parser;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.IncorrectCommand;
+import seedu.address.logic.commands.SearchCommand;
+
+public class SearchParser {
+	private static final Pattern CMD_PATTERN =
+            Pattern.compile("^(?<keyword>)"+"$");
+
+
+	public Command parse(String args){
+    	final Matcher matcher = CMD_PATTERN.matcher(args.trim());
```

Comment:
Seems like this would not properly split the string? Probably should involve a `String.split()`?